test/lint.rb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+require "minitest/autorun"
+
+$LOAD_PATH.unshift File.expand_path(File.join(File.dirname($0), "..", "lib"))
+require "dokumi"
+
+class TestFindbugs < Minitest::Test
+  def build_script
+    File.expand_path(File.join(File.dirname($0), "build", "dokumi-test-android-lint.rb"))
+  end
+
+  def test_static_analysis
+    issues = Dokumi::Command.review("github.com", "tatsuhama", "DokumiLint", 1, skip_comment_creation: true, build_script: build_script)
+    
+    assert_equal 2, issues.length
+    issues.each do |issue|
+      assert_equal :lint, issue[:tool]
```

Comment:
According to our styleguide you should use `{}` syntax on the statement in one line.
https://github.com/cookpad/styleguide/blob/master/ruby.en.md#method-calls

`issues.each { |issue| assert_equal :lint, issue[:tool] }`